tic_tac_toe/game.py

```diff
@@ -63,22 +82,63 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    diagonals = [ 
+                    ((2, 0), (1, 1), (0, 2)),
+                    ((0, 0), (1, 1), (2, 2))
+                ]
+    columns = [
+                    ((0, 0), (1, 0), (2, 0)),
+                    ((0, 1), (1, 1), (2, 1)),
+                    ((0, 2), (1, 2), (2, 2))
+              ]
+    rows = [         
+                    ((0, 0), (0, 1), (0, 2)),
+                    ((1, 0), (1, 1), (1, 2)),
+                    ((2, 0), (2, 1), (2, 2))
+           ]
```

Comment:
Can you explain your thinking behind dividing the winning combinations up like this? I can see it aids readibility, but is there any other benefit of doing it this way rather than just in one construct. Also, did you choose tuples over lists for speed and efficiency?